modules/Makefile.circleci

```diff
@@ -42,11 +42,13 @@ circle\:release:
 ## Deploy to kubernetes after obtaining an exclusive lock
 circle\:deploy-kubernetes:
 	$(call assert,CIRCLE_BRANCH)
+	$(SELF) circle:tag kubernetes:info
```

Comment:
Note: these targets are needed both when we need a lock and not, so I moved them up